cmd/version.go

```diff
@@ -0,0 +1,49 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/spf13/cobra"
+)
+
+var Run_number string
```

Comment:
@Daniel-GrunbergerCA , the variable name is not "go like". It should be `runNumber`, though I would call it `buildNumber`